script/module/CellMap.js

```diff
@@ -0,0 +1,112 @@
+import Cell from 'module/Cell';
+
+export default class CellMap extends Array {
```

Comment:
Heftig hoor om `Array` te extenden 😂

Reply:
Haha ja maar ik werd een beetje gek van:

```
CellMap {
    constructor(cells) {
        this.cells = cells;
    }
```

```
let cellMap = CellMap.createWithSize({witdh: 3, height: 3});
cellMap.cells[1][3]...
```

Leek me dan logischer om gewoon te zeggen: dit ding is een Iterable met een zooi functies.